services/core/SQLHistorian/tests/test_sqlitehistorian_unit.py

```diff
@@ -1,8 +1,8 @@
 import os
-from shutil import rmtree
 import subprocess
-
 import pytest
+
```

Comment:
please reorganize these imports.  the order should be built-ins, add-ons, our stuff (generally)

easy to do with pycharm.  code menu I believe